doc/python/configuration-options.md

```diff
@@ -132,10 +305,56 @@ fig = go.Figure(go.Bar(
 
 fig.update_layout(xaxis = {'type': 'date'})
 
-fig.show(config = {'doubleClickDelay': 1000})
+fig.show(config=config)
 ```
 
-#### Reference
+#### Configuring Figures in Dash Apps
```

Comment:
this should probably be reworked a bit once https://github.com/plotly/documentation/issues/1654 happens.